### PR TITLE
Release 2022.02.1

### PR DESCRIPTION
**Pull request by jsignell**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes https://github.com/conda-forge/distributed-feedstock/issues/198

### CODE REVIEW

**Comment by jsignell** [comment]
@conda-forge-admin, please rerender

**Comment by jsignell** [comment]
@jakirkham I tried to set a min python in the distributed recipe and I think that was maybe a bad idea. Is there a right way to do that or should I just not?

**Review by jrbourbeau — COMMENTED** [review]
Thanks @jsignell. We should be able to make `distributed` a `noarch` package now. That should help with the linting issue

**Comment by jsignell** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,26 +7,24 @@ package:
...
     - dask-worker = distributed.cli.dask_worker:go
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.8
     - pip
```
Comment (jakirkham) [inline recipe/meta.yaml:25]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -39,8 +37,7 @@ requirements:
     - sortedcontainers !=2.0.0,!=2.0.1
     - tblib >=1.6.0
     - toolz >=0.8.2
-    - tornado >=5  # [py<38]
-    - tornado >=6.0.3  # [py>=38]
+    - tornado >=6.0.3
     - zict >=0.1.3
     - setuptools <60.0.0
```
Comment (jakirkham) [inline recipe/meta.yaml:43]:
```suggestion
```

**Review by jakirkham — COMMENTED** [review]
Yep James suggestions sound reasonable. Looks like those are already here

Had a few minor whitespace nits that I thought we could address at the same time

**Comment by jsignell** [comment]
@jakirkham it looks like this is failing linting.

**Comment by jakirkham** [comment]
Yeah we just have to wait for CDN. Lint seems ok AFAICT

**Comment by jsignell** [comment]
> FWIW there's [a command to restart CI](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci)

ooooh :+1:

**Pull request merged**